test/a11y.html

```diff
@@ -74,7 +74,7 @@
         assert.isTrue(focusSpy.called);
       });
 
-      a11ySuite('trivialSlider');
+      // a11ySuite('trivialSlider');
```

Comment:
still broken? can you double check, pls? (some changes were merged in a11ySuite)